qos-service-impl/src/main/java/com/palantir/atlasdb/qos/config/ThrottlingStrategy.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.qos.config;
+
+import java.util.List;
+
+public interface ThrottlingStrategy {
+
+    enum ThrottlingStrategies {
```

Comment:
this might be better as just a static final variable (`ThrottlingStrategy SIMPLE = new SimpleThrottlingStrategy();`)

Reply:
In the future, we will want to add more throttling strategies. Making this a static final will love the configurability, we can take a string as the config parameter and map to the right one but an enum feels cleaner.

Reply:
ok, we need to fix the fact that the `SimpleThrottlingStategy` is stateful though - `getThrottlingStrategy` should probably be `createThrottlingStrategy` and should return a new instance